fvm/src/gas/price_list.rs

```diff
@@ -323,24 +323,18 @@ pub(crate) struct StepCost(Vec<Step>);
 
 #[derive(Clone, Debug, Copy, PartialEq, Eq)]
 pub(crate) struct Step {
-    start: i64,
+    start: u64,
     cost: Gas,
 }
 
 impl StepCost {
-    pub(crate) fn lookup(&self, x: i64) -> Gas {
-        let mut i: i64 = 0;
-        while i < self.0.len() as i64 {
-            if self.0[i as usize].start > x {
-                break;
-            }
-            i += 1;
-        }
-        i -= 1;
-        if i < 0 {
-            return Gas::zero();
-        }
-        self.0[i as usize].cost
+    pub(crate) fn lookup(&self, x: u64) -> Gas {
```

Comment:
TODO: also write a test. I changed this because it's easier to prove that the new code never panics or anything.